### ISSUE TITLE
Pipes creates fps lags

### ISSUE THREAD

**Issue by Biggbosscurser** [DaedalusGame/EmbersRekindled#34]
I have been using Embers 0.230 and upgraded to Rekindled version 1.4hotfix. i had serious fps lags when i have visual contact with my ember machines. I have everything Embers in my basement and i am in the same chunk as my machines most of the time without any visuals and they work alright, when i start to see them i had problems. further experimantations showed me that it was because of my fluid extractor from the fluid vessel to pressure refinery.  I downdate my embers back to 0.230 and everything works fine.

**Comment by gazpachoking** [comment]
Yep, having this same problem as well. A fluid extractor in my general field of view causes my fps to go down from 60 to 20 or less.

**Comment by Psion77** [comment]
I can also confirm this as well. I made a back-up of my world and deleted all Embers pipes. I had a problem area where I was getting 15-20 FPS that climbed back up to 45-55 when the pipes were gone. I was able to get back to 60 if I also deleted Bibliocraft chests in the area. Using 1.5hotfix.

**Comment by DaedalusGame** [comment]
potentially fixed in 1.6

**Comment by Psion77** [comment]
I still see the lag with 1.6. Note that I don't see the lag when I am right next to the pipes. Instead, I see it when I am within render distance and am looking in a direction where they would be. I can't actually see them from these locations because they are all in the basement of my tower. 

https://imgur.com/a/0unxQ9E

In the first image, you can see only a bit of lag when I am right next to the pipes. 

In the second image, you can see that there is a lot of lag from the top of my tower.  The issue seems to exist when you are near the pipes, but can't actually see them, at least in my case. 

If I turn around, I have 60fps (capped).

**Comment by Psion77** [comment]
Update: I turned the setting UseVBOs from on to off. This seems to have largely fixed the problem. I don't know exactly what Vertex Buffer Objects actually are, but I'm guessing that they are an Optifine thing, and that this might be another case of Optifine breaking a mod. 

Even stranger, when I turned VBOs back on, the problem stayed fixed. Perhaps I will have to do this again next time I load this area. 

At least this seems to be pointing in the right direction as to the true cause.
